spack/configs/summit/packages.yaml

```diff
@@ -1,25 +1,61 @@
 packages:
-  openssl:
-    paths:
-      openssl@1.0.2p: /usr
-    buildable: False
 
+  # concretization preferences
+  all:
+    providers:
+      mpi: [spectrum-mpi]
+      blas: [netlib-lapack]
+      lapack: [netlib-lapack]
+      scalapack: [netlib-scalapack]
+
+  gene:
+    compiler: [gcc@8.1.1]
+
+  xgc1:
+    compiler: [pgi@19.9]
+
+  libfabric:
+    variants: fabrics=verbs
+
+  adios:
+    variants: +fortran +hdf5 +mpi
+
+  adios2:
+    # Note: +python is not currently work with python3.7.0anaconda and gcc 8.1.1
+    variants: +fortran +hdf5 +mpi
+
+  dataspaces:
+    version: [develop]
+    variants: +fpic maxdims=10 network-type=infiniband +dimes
```

Comment:
So I'm about unsure about whether this is a good idea, because this is setting defaults that are global in my account even though these are (I think) specific to gene/xgc1.  But xgc1/gene already specify what they want from adios, so if prereqs for them already get installed correctly, and so the above isn't really used -- it's used if I install adios{,2] directly, and who knows what my preferences are for that ;)

(I'm less sure about dataspaces -- maybe some of those parameters are summit-specific, so in that case, it 'd make sense to have that in there.)

Reply:
Good point, adios/adios2 shouldn't need to be there, I'll remove them.